UE/Application/States/ConnectedState.cpp

```diff
@@ -113,5 +113,12 @@ void ConnectedState::handleUnknownRecipient()
     const auto index = gsl::narrow_cast<gsl::index>(numberOfMessages) - 1;
     context.smsDb.setMessageState(index, SmsState::Failed);
 }
+void ConnectedState::handleReceiveCallDrop()
+{
+    //todo #check
```

Comment:
I think that we should inform user, that correspondent reject our call. Use `alertUser` function from `userPort`.

Example:
`context.user.alertUser("Call rejected");`